xclim/indices/_threshold.py

```diff
@@ -717,6 +719,68 @@ def maximum_consecutive_tx_days(
     return group.apply(rl.longest_run, dim="time")
 
 
+@declare_units("[area]", sic="[]", area="[area]", thresh="[]")
+def sea_ice_area(sic, area, thresh="15 pct"):
+    """Return the total sea ice area.
+
+    Sea ice area measures the total sea ice covered area where sea ice concentration is above a threshold,
+    usually set to 15%.
+
+    Parameters
+    ----------
+    sic : xarray.DataArray
+      Sea ice concentration [0,1].
+    area : xarray.DataArray
+      Grid cell area [m²]
+    thresh : str
+      Minimum sea ice concentration for a grid cell to contribute to the sea ice extent.
+
+    Returns
+    -------
+    Sea ice area [m²].
+
+    Notes
+    -----
+    To compute sea ice area over a given region, first mask the sea ice concentration.
```

Comment:
line 744 - aren't we doing this for them in line 748?

Reply:
Line 748 masks grid cells where sic is below a threshold.

Reply:
> Line 748 masks grid cells where sic is below a threshold.

Ok. We are referring to spatial subsets (more or less) in this note right.  Could maybe change to something like : To compute sea ice area over a subregion, first mask or subset the input sea ice concentration data.

Reply:
Done.